date-time/src/commonMain/kotlin/com/splendo/kaluga/datetime/timer/RecurringTimer.kt

```diff
@@ -52,7 +52,8 @@ class RecurringTimer(
     coroutineScope: CoroutineScope = MainScope()
 ) : ControllableTimer {
     private val stateRepo = TimerStateRepo(duration, interval, timeSource, delayFunction, coroutineScope)
-    override val state: StateFlow<Timer.State> = stateRepo.stateFlow
+    override val state: Flow<Timer.State> = stateRepo.stateFlow.map { it.timerState }
+    override val defaultState: Timer.State = stateRepo.stateFlow.value.timerState
```

Comment:
What is default state? Now it's some initial unknown state which depends on repo initialisation

Reply:
I don't think we need a "default state", it's just `current` or `currentState`?

Reply:
Renamed to currentState. Its to account for the initial state mentioned in the comment above

Reply:
good for me now, if you agree can you resolve comments @avdyushin ?

Reply:
For current state it should be `get()`?

Reply:
Indeed, it was cache in github, resolved!